hugo/config.toml

```diff
@@ -6,7 +6,8 @@ publishDir = "../public"
 disableKinds = ["taxonomy","taxonomyTerm"]
 
 [permalinks]
-  blog = "/blog/:title/"
+  news = "/news/:title/"
+  blog = "/news/:title/"
```

Comment:
I presume this sets up a duplicate system rather than a new one with the old one redirecting?

I think blog should be an alias. https://gohugo.io/content-management/urls/

Reply:
> question: I presume this sets up a duplicate system rather than a new one with the old one redirecting?

I don't know if I understand your question correctly, but:
- It is intended to use `/news` only, so no duplicate system. I have also changed all links on the website to `/news`.
- this `blog = "/news/:title/"` is only intended to redirect as suggested by @Krzmbrzl 

If you prefer other methods for a general redirect, you are free to suggest or implement them.

> I think blog should be an alias. https://gohugo.io/content-management/urls/

Regarding the negative answer above, this is not the ideal solution.

Reply:
I just tested it and as I suspected /blog/ is still accessible as pages. It is not redirects to /news/.

Reply:
That is bad. I guess we need a different solution then.